packages/design-system/src/components/Dialog/useDialogAnalytics.ts

```diff
@@ -31,6 +31,7 @@ export function useDialogAnalytics({
       event_type: EventType.UI_INTERACTION,
       event_category: EventCategory.UI_COMPONENTS,
       event_label: eventHeadingText,
+      event_extension: 'Design system integration',
```

Comment:
This string is repeated in several places. How would you feel about making this a constant?

Reply:
Yeah not a bad idea since it's something we want to remain the same across all events by default.  I've just pushed those changes.

Reply:
whoops not ready yet

Reply:
Ok good to go 👍🏻